prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -278,22 +281,32 @@ public override async Task<bool> ProcessRule(Enrollee enrollee)
                 if (paperEnrolleeMatchId == -1)
                 {
                     enrollee.AddReasonToCurrentStatus(StatusReasonType.PaperEnrolmentMismatch, $"User-Provided GPID: {potentialPaperEnrolleeGpid}");
+                    enrollee.AddReasonToCurrentStatus(StatusReasonType.PossiblePaperEnrolmentMatch, $"birthdate matches enrolment(s): {paperEnrolleeIdsAsString}.");
```

Comment:
Need to capitalize the "B" and probably remove the period, Don't think we use them anywhere. 

On that note might also want to change the config for the reason from
"Possible match with paper enrolment" -> "Possible match with paper enrolment(s)" 
to reflect the multiple matching.

Reply:
updated